pkgs/development/python-modules/buildbot/plugins.nix

```diff
@@ -6,11 +6,13 @@
     inherit (buildbot-pkg) version;
 
     src = fetchPypi {
-      inherit pname version;
-      sha256 = "0l4kyxh62y86cw97101gjs42n1sdw1n18cgh6mm337gzjn42nv3x";
+      pname = "buildbot_www";
+      inherit version format;
+      python = "py3";
+      sha256 = "1wbkkv046h6yb0fnh39ypva4x809dphjwvzv43dpgpnshlya4aws";
     };
 
-    buildInputs = [ buildbot buildbot-pkg mock ];
+    format = "wheel";
```

Comment:
avoid wheels when possible

Reply:
The only way I can see to avoid a wheel is to patch the package's setuptools script.

Reply:
A comment explaining that should be added.
How about using `pipInstallFlags = [ "--ignore-dependencies" ];` instead?